backend/package-lock.json

```diff
@@ -0,0 +1,1044 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
```

Comment:
Remove this file (package-lock) as we don't use "npm" in our project

Reply:
Ok, no problem